storybook/src/brand/design-tokens/grid.docs.mdx

```diff
@@ -136,17 +136,17 @@ In these 3 situations, the dimensions are as follows.
 
 | Device Class | Reference Width | Number of columns | Column width | Gap width | Margin width | Grid width |
 | :----------- | --------------: | ----------------: | -----------: | --------: | -----------: | ---------: |
-| phone        |             320 |                 4 |         39.5 |        36 |           27 |        266 |
-| tablet       |             832 |                 8 |         54.5 |        44 |           44 |        744 |
-| desktop      |            1600 |                12 |           68 |        56 |           84 |       1432 |
+| phone        |             320 |                 4 |           41 |        36 |           24 |        272 |
+| tablet       |             880 |                 8 |           56 |        48 |           48 |        784 |
+| desktop      |            1440 |                12 |           50 |        60 |           90 |       1260 |
 
 #### Compact
 
 | Device Class | Reference Width | Number of columns | Menu bar width | Column width | Gap width | Margin width | Grid width |
 | :----------- | --------------: | ----------------: | -------------: | -----------: | --------: | -----------: | ---------: |
-| phone        |             320 |                 4 |              0 |           57 |        20 |           16 |        288 |
-| tablet       |             832 |                 8 |              0 |         74.3 |      26.4 |         26.4 |      779.2 |
-| desktop      |            1600 |                12 |            112 |           98 |        36 |           48 |       1392 |
+| phone        |             320 |                 4 |              0 |           54 |        24 |           16 |        288 |
+| tablet       |             880 |                 8 |              0 |         76.3 |        30 |           30 |        820 |
+| desktop      |            1920 |                12 |            108 |          110 |        36 |           48 |       1716 |
```

Comment:
This might change if we deliver the application layout component with the sidebar navigation.